docs/blog/2020-community-update.rst

```diff
@@ -0,0 +1,116 @@
+.. post:: Sep 7, 2020
+   :tags: stats, newsletter
+   :author: Eric Holscher
+
+Write the Docs 2020 Community Update
```

Comment:
Not sure why the "2020" is here, this feels like a mid-year update. IMO we can remove it and still get the message across